Sources/TitleView.swift

```diff
@@ -8,7 +8,7 @@ open class TitleView: UIView {
 
   // MARK: - Initialization
 
-  public init?(navigationController: UINavigationController, title: String, items: [String]) {
+	public init?(navigationController: UINavigationController, title: String, items: [String], initialIndex: Int? = nil) {
```

Comment:
You can make it non optional, like `initialIndex: Int = 0`

Reply:
Excellent observation!